src/Search/Gearman/Command/QueueCommand.php

```diff
@@ -120,23 +125,29 @@ public function checkStatus(InputInterface $input, LoggerInterface $logger)
 
     public function loop(InputInterface $input, LoggerInterface $logger)
     {
+        $logger->info('Loop start... [');
         $topic = $input->getArgument('topic');
         $timeout = $input->getOption('queue-timeout');
-        $logger->info('Hello queue. topic: '.$topic);
+        $logger->info('-> Hello queue. topic: '.$topic);
```

Comment:
For parameterizable logging in general, I suggest to use the $context argument https://github.com/Seldaek/monolog/blob/master/doc/01-usage.md#using-the-logging-context which would make this easier to search from log aggregators